icefoganalytics/travel-authorization
# Issue 90: Pre-Work Component Naming Standardization

# Context

In preparation of https://github.com/icefoganalytics/travel-authorization/issues/90, I now have enough components to need a standard for naming them.

The standard I've adopted is:
`<action/role><object><thing>`
e.g.
- `web/src/modules/travel-authorizations/pages/ManageTravelAuthorizationsPage.vue`
- `web/src/modules/travel-authorizations/pages/EditMyTravelAuthorizationEstimatePage.vue`
- `web/src/modules/travel-authorizations/pages/MyTravelAuthorizationsPage.vue` - no action because there is only one of these and it does read/edit both.

I also added a couple readme-s.

# Testing instructions

1. Boot the app via `dev up`.
2. Log in to the app at http://localhost:8080
3. Check that you can go to the My Travel Request page via the top drop down nav.
4. Check that you can create travel authorizations.
5. Check that you can view travel authorizations that have been submitted.
6. Check that you can go to the "Manager View" from the top drop down nav.
